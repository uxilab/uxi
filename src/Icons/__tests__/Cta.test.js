import React from 'react';
import Cta from '../Cta';

describe('Icon : <Cta />', () => {
  it('match snapshot', () => {
    expect(shallow(<Cta />)).toMatchSnapshot();
  });
});

