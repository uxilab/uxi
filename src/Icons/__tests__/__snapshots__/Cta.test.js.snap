// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon : <Cta /> match snapshot 1`] = `
<SvgIcon
  color="currentColor"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  size={24}
  style={Object {}}
  viewBox="0 0 24 24"
>
  <svg
    focusable="false"
    height="24px"
    viewBox="0 0 9.93 16"
    width="24px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Layer 2"
      id="Layer_2"
    >
      <g
        data-name="Layer 1"
        id="Layer_1-2"
      >
        <path
          d="M8.69,8.82a1.11,1.11,0,0,0-.53.12,1.23,1.23,0,0,0-1.59-.48,1.25,1.25,0,0,0-1.43-.55V5.74a1.18,1.18,0,0,0-.37-.87,1.24,1.24,0,0,0-2.12.87v5l-.43-.43A1.28,1.28,0,0,0,.64,10.2a1.22,1.22,0,0,0-.54.86,1.33,1.33,0,0,0,.27,1l2.18,2.63A3.61,3.61,0,0,0,5.35,16H6.71a3.21,3.21,0,0,0,3.22-3.22V10.06A1.23,1.23,0,0,0,8.69,8.82ZM3.56,11.39V5.74a.31.31,0,0,1,.09-.25.37.37,0,0,1,.5,0,.33.33,0,0,1,.1.25V9.62a.44.44,0,0,0,.44.45.45.45,0,0,0,.45-.45V9.1a.36.36,0,0,1,.36-.36.37.37,0,0,1,.37.36v1a.44.44,0,0,0,.44.45.45.45,0,0,0,.45-.45V9.58a.36.36,0,0,1,.72,0v1a.44.44,0,0,0,.45.44.43.43,0,0,0,.44-.44v-.53a.36.36,0,0,1,.71,0v2.69a2.45,2.45,0,0,1-.71,1.66,2.3,2.3,0,0,1-1.66.69H5.35a2.76,2.76,0,0,1-2.11-1L1.06,11.47A.33.33,0,0,1,1,11.19a.35.35,0,0,1,.15-.26.42.42,0,0,1,.21-.06.4.4,0,0,1,.27.11l.87.87a.63.63,0,0,0,.69.13A.66.66,0,0,0,3.56,11.39Z"
        />
        <path
          d="M2.18,3.86a.49.49,0,0,1-.34-.14L.14,2a.48.48,0,0,1,0-.68.48.48,0,0,1,.68,0L2.52,3a.5.5,0,0,1,0,.68A.51.51,0,0,1,2.18,3.86Z"
        />
        <path
          d="M3.89,3.36a.48.48,0,0,1-.48-.48V.48A.48.48,0,0,1,3.89,0a.47.47,0,0,1,.48.48v2.4A.47.47,0,0,1,3.89,3.36Z"
        />
        <path
          d="M5.53,3.86a.49.49,0,0,1-.34-.14.48.48,0,0,1,0-.68l1.7-1.7a.48.48,0,0,1,.68,0,.5.5,0,0,1,0,.68l-1.7,1.7A.51.51,0,0,1,5.53,3.86Z"
        />
      </g>
    </g>
  </svg>
</SvgIcon>
`;
